src/elm/Page/Dashboard/Analysis.elm

```diff
@@ -105,22 +122,19 @@ view ({ shared } as loggedIn) model =
         t : String -> String
         t =
             I18Next.t shared.translations
-
-        text_ s =
-            text (t s)
     in
     case model.status of
         Loading ->
             Page.fullPageLoading
 
-        Loaded maybeProfile f claims pageInfo ->
+        Loaded claims pageInfo ->
             div []
-                [ Page.viewHeader loggedIn (t "dashboard.all_analysis.title") Route.Dashboard
+                [ Page.viewHeader loggedIn (t "all_analysis.title") Route.Dashboard
                 , div [ class "container mx-auto px-4 mb-10" ]
-                    [ --viewAutoCompleteProfile loggedIn model maybeProfile
-                      -- , text "filtro do estado do claim"
-                      div [ class "flex flex-wrap -mx-2" ]
-                        (List.map (viewClaim loggedIn f) claims)
+                    [ viewFilters loggedIn model
+                    , div
+                        [ class "flex flex-wrap -mx-2" ]
+                        (List.map (viewClaim loggedIn) claims)
```

Comment:
It's a bit confusing to see an empty space when there are no claims that were found according to the selected criteria. It's better to tell the user explicitly that there are no claims were found:

![Screenshot 2020-05-26 at 14 33 39](https://user-images.githubusercontent.com/140053/82896055-f4705980-9f5d-11ea-9a3d-14fdea5272ea.png)

```elm
(if List.isEmpty claims then
    [ p [ class "w-full text-center" ] [ text "No claims were found" ] ]

 else
    List.map (viewClaim loggedIn) claims
)
```

Reply:
agreed! waiting on @heltonlr to send an empty list state for this (maybe it will be the same on dashboard but lets wait)

Reply:
Here @lucca65 , sorry for the delay:
https://www.figma.com/file/nEPG6Dkns8hDVCudVUTEzc/Dashboard?node-id=1316%3A22031

Reply:
Done!